src/components/TooltipController/TooltipControllerTypes.d.ts

```diff
@@ -26,3 +26,18 @@ export interface ITooltipController {
   isOpen?: boolean
   setIsOpen?: (value: boolean) => void
 }
+
+declare module 'react' {
+  interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
+    'data-tooltip-place'?: PlacesType
+    'data-tooltip-content'?: string | number
+    'data-tooltip-html'?: string
+    'data-tooltip-variant'?: VariantType
+    'data-tooltip-offset'?: number
+    'data-tooltip-wrapper'?: WrapperType
+    'data-tooltip-events'?: EventsType[]
+    'data-tooltip-position-strategy'?: PositionStrategy
+    'data-tooltip-delay-show'?: number
+    'data-tooltip-delay-hide'?: number
+  }
+}
```

Comment:
this will be relevant after figuring out how to emit all types to the build `react-tooltip.d.ts` file.

Autocomplete doesn't work too well with attributes with a `-` in them, but this should enable autocomplete for the values, and throw a compile error when using invalid values.

For example, an element such as `<div data-tooltip-place="botom" />` would not compile (`botom` is not a valid value for `place`)

Reply:
Oh, got it!